src/DynamoPublish/Views/PublishView.xaml.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Dynamo.Publish.ViewModels;
+using Dynamo.Wpf.Authentication;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Forms;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace Dynamo.Publish.Views
+{
+    /// <summary>
+    /// Interaction logic for PublishView.xaml
+    /// </summary>
+    public partial class PublishView : Window
+    {
+        internal PublishViewModel ViewModel
+        {
+            get
+            {
+                return DataContext as PublishViewModel;
+            }
+        }
+
+        public PublishView(PublishViewModel viewModel)
+        {
+            InitializeComponent();
+            DataContext = viewModel;
+
+            viewModel.RequestLogin += CreateLoginService;
+        }
+
+        private void CreateLoginService()
+        {
+            ViewModel.LoginService = new LoginService(this, new WindowsFormsSynchronizationContext());
+        }
+
+        private void OnPublishViewClosing(object sender, CancelEventArgs e)
+        {
+            ViewModel.RequestLogin -= CreateLoginService;
+        }
```

Comment:
Like I mentioned above, we should not create the `LoginService` here.  Dynamo will handle creating it.